src/balancerd/src/lib.rs

```diff
@@ -209,6 +217,23 @@ impl BalancerService {
             cfg.launchdarkly_sdk_key.as_deref(),
             cfg.config_sync_timeout,
             cfg.config_sync_loop_interval,
+            move |updates, configs| {
+                if [
+                    LOGGING_FILTER.name(),
+                    OPENTELEMETRY_FILTER.name(),
+                    LOGGING_FILTER_DEFAULTS.name(),
+                    OPENTELEMETRY_FILTER_DEFAULTS.name(),
+                    SENTRY_FILTERS.name(),
+                ]
+                .into_iter()
+                .any(|name| updates.updates.contains_key(name))
```

Comment:
Seems worth pulling out a `has_tracing_config_change` function and sticking it next to `tracing_config`, just to increase the odds that if you change one you remember to change the other.